trajectory_msgs/JointTrajectory jointTrajectory
nav_msgs/Path pathTrajectory
std_msgs/String controller
